app/views/static_pages/_book.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="col-sm-3" >
+  <div class="product-card">
+    <div class="badge"><%= t(".new") %></div>
+    <div class="product-tumb">
+      <img src="https://www.dtv-ebook.com/images/files_2/2020/ban-that-ba-dao!-jen-sincero.jpg" alt="">
+    </div>
+    <div class="product-details">
+      <span class="product-catagory"><%= book.author.name %> | <%= book.category.name %></span>
```

Comment:
`book.author.name` tìm hiểu về `delegate in rails` nhé

Reply:
oke chị đã fix ạ